tests/rest/client/test_relations.py

```diff
@@ -1636,9 +1638,9 @@ def test_redact_relation_thread(self) -> None:
         ##################################################
         self.assertEquals(self._get_related_events(), list(reversed(thread_replies)))
```

Comment:
It looks like `assertEquals` is also being removed?

Reply:
Fixed by you in #16110 :)